sr/robot3/astoria.py

```diff
@@ -28,126 +31,197 @@ class GetMetadataResult(NamedTuple):
     usb_path: Path
 
 
-class GetMetadataConsumer(StateConsumer):
-    """Astoria consumer to fetch metadata."""
+class GetMetadataConsumer():
```

Comment:
```suggestion
class GetMetadataConsumer:
```

Brackets no longer necessary